src/lug/System/Win32/PathImplWin32.cpp

```diff
@@ -0,0 +1,76 @@
+#include <lug/System/Win32/PathImplWin32.hpp>

```

Comment:
I think that you can comment at the beginning of each function to indicate the attended path.
Like `DRIVE:/Users/{LOGIN}/`